iotilecore/iotile/core/hw/reports/signed_list_format.py

```diff
@@ -4,12 +4,18 @@
 import struct
 import hashlib
 import hmac
+from enum import IntEnum
 from .report import IOTileReport, IOTileReading
 from iotile.core.utilities.packed import unpack
 from iotile.core.exceptions import NotFoundError, ExternalError
 from iotile.core.hw.auth.auth_provider import AuthProvider
 from iotile.core.hw.auth.auth_chain import ChainedAuthProvider
 
+class ReportSignatureFlags(IntEnum):
+    """Types of stream report signeature"""
+    SIGNED_WITH_HASH = 0
```

Comment:
Typo. "signeature" should be "signature".

Reply:
Got it